asv/environment.py

```diff
@@ -275,18 +290,18 @@ def create(self):
                     pass
 
             try:
-                self._setup()
+                self.setup()
             except:
                 log.error("Failure creating environment for {0}".format(self.name))
                 if os.path.exists(self._path):
-                    shutil.rmtree(self._path)
+                    util.long_path_rmtree(self._path)
                 raise
 
         self.save_info_file(self._path)
 
         self._is_setup = True
 
-    def _setup(self):
+    def setup(self):
```

Comment:
Why the change to public?  The original intention here was that this should only be called from `create`...